src/Sulu/Bundle/SecurityBundle/Resources/js/views/Form/tests/toolbarActions/LockUserToolbarAction.test.js

```diff
@@ -0,0 +1,256 @@
+// @flow
+import ResourceStore from 'sulu-admin-bundle/stores/ResourceStore';
+import Router from 'sulu-admin-bundle/services/Router';
+import Form from 'sulu-admin-bundle/views/Form/Form';
+import {ResourceFormStore} from 'sulu-admin-bundle/containers/Form';
+import {ResourceRequester} from 'sulu-admin-bundle/services';
+import LockUserToolbarAction from '../../toolbarActions/LockUserToolbarAction';
+
+jest.mock('sulu-admin-bundle/utils', () => ({
+    translate: jest.fn((key) => key),
+}));
+
+jest.mock('sulu-admin-bundle/stores/ResourceStore', () => jest.fn(function() {
+    this.data = {};
+}));
+
+jest.mock('sulu-admin-bundle/containers/Form', () => ({
+    ResourceFormStore: class {
+        resourceStore;
+
+        set = jest.fn();
+
+        constructor(resourceStore) {
+            this.resourceStore = resourceStore;
+        }
+
+        get id() {
+            return this.resourceStore.id;
+        }
+
+        get data() {
+            return this.resourceStore.data;
+        }
+
+        get locale() {
+            return this.resourceStore.locale;
+        }
+
+        get loading() {
+            return this.resourceStore.loading;
+        }
+    },
+}));
+
+jest.mock('sulu-admin-bundle/services/Router', () => jest.fn(function() {}));
```

Comment:
Just a small note, you don't have to pass an empty function, just `jest.fn()` would do just fine :slightly_smiling_face: